extensions/wikia/AffiliateService/js/tracker.js

```diff
@@ -8,11 +8,11 @@ define('ext.wikia.AffiliateService.tracker', [
 	var defaultOptions = {
 		campaignId: '',
 		categoryId: '',
-		extraTracking: {},
+		extraTracking: [],
 	}
 
 	function track(action, label, options) {
-		var currentOptions = $.extend(defaultOptions, options || {});
+		var currentOptions = $.extend({}, defaultOptions, options || {});
 
 		// convert extra tracking options
 		var extraTracking = {};
```

Comment:
This is a little bit confusing with `defaultOptions.eventTracking` up above being an array and the same variable name here as an object.